### PR TITLE
Screwdriver changes to add new drivers

### PR DESCRIPTION
**Pull request by tenplus1**
A few screwdriver changes so that mod makers can add new screwdrivers with different wear levels.  Tried and tested with new diamond screwdriver :)

### CODE REVIEW

**Comment by 0-afflatus** [comment]
Nice idea

**Comment by tenplus1** [comment]
Behold, the ultimate in screwdriver technology...  the Diamond Screwdriver...  *angels sing*

**Comment by C1ffisme** [comment]
:+1:

File: mods/screwdriver/init.lua [hunk 1]
```diff
@@ -18,10 +18,9 @@ screwdriver.rotate_simple = function(pos, node, user, mode, new_param2)
 		return false
 	end
 end
-local USES = 200
 
 -- Handles rotation
-local function screwdriver_handler(itemstack, user, pointed_thing, mode)
+screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
```
Comment (kaeza) [inline mods/screwdriver/init.lua]: `function screwdriver.handler` pls

**Comment by kaeza** [comment]
:+1:

>angels sing

The original code was done by a real bad one, IIRC. :P

File: mods/screwdriver/init.lua (hunk 1 above)
Comment (paramat) [inline mods/screwdriver/init.lua]: Agrred.

**Comment by Ekdohibs** [comment]
:+1: if the 'uses' nil check is added.

**Comment by tenplus1** [comment]
moved to https://github.com/minetest/minetest_game/pull/893

**Pull request closed without merging**